src/App.js

```diff
@@ -1,9 +1,9 @@
+import Calculator from './components/calculator';
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1 className="header-title">Hello React</h1>
-      </header>
+      <Calculator />
     </div>
   );
```

Comment:
- [ ] Kindly convert the `App` component to class-based instead of function based as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).).

Reply:
owwww I see, thank you very much. I will walk through it.